src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -483,7 +484,7 @@ private string InnerGenerateFile()
 					{
 						var generationRunFileInfo = runFileInfo.GetRunFileInfo(_fileUniqueId);
 
-						writer.AppendLineInvariantIndented("{0}", generationRunFileInfo.ComponentCode);
+						writer.AppendLineIndented(generationRunFileInfo.ComponentCode);
```

Comment:
This needs to be done the existing way, where otherwise component code containing `{}` will fail.

Reply:
@jeromelaban Can you clarify, I don't see why this can fail. The `AppendLineIndented` call will call `AppendIndented` which calls `StringBuilder.Append`:

https://github.com/unoplatform/uno/blob/340d1474388a4a75823d66fadddb6b0e32338e2e/src/Uno.Foundation/Uno.Core.Extensions/Uno.Core.Extensions.Compatibility/Extensions/IndentedStringBuilder.cs#L77-L81

![image](https://user-images.githubusercontent.com/31348972/221497637-460d0d98-ab98-4630-bd46-6de6852af793.png)

There is `AppendFormat` involved in `AppendLineIndented` so it should be good?

Reply:
Ah, I missed that you removed the formatting :) Yes, it's good.